Assets/Scripts/UI/Pickup Pup/PPUIController.cs

```diff
@@ -3,22 +3,31 @@
  * Description: Controls a UI
  */
 
-public class PPUIController : MonoBehaviourExtended {
+public class PPUIController : MonoBehaviourExtended 
+{
 
 	protected PPSceneController scene;
     protected PPGameController ppGameController;
 
-    protected override void FetchReferences () {
-		base.FetchReferences ();
+	#region MonoBehaviourExtended Overrides
+
+    protected override void fetchReferences() 
+	{
+		base.fetchReferences();
 		scene = PPSceneController.Instance;
         ppGameController = (PPGameController) PPGameController.Instance;
     }
 
-    public void LoadStart () {
+	#endregion
+
+    public void LoadStart() 
+	{
 		scene.LoadStart();
 	}
 
-	public void LoadHome () {
+	public void LoadHome()
+	{
 		scene.LoadHome();
 	}
-}
\ No newline at end of file
+
+}
```

Comment:
Must be a tabs vs. spaces problem. But if it doesn't show up in the editor, then maybe it doesn't matter.